src/catchup/CatchupManagerImpl.cpp

```diff
@@ -64,7 +64,7 @@ CatchupManagerImpl::processLedger(LedgerCloseData const& ledgerData)
     ZoneScoped;
     if (mCatchupWork && mCatchupWork->isDone())
     {
-        trimAndReset();
```

Comment:
just place the reset code here, this function is not used anywhere else

Reply:
Done.